bittide-instances/src/Bittide/Instances/Hitl/FullMeshSwCc.hs

```diff
@@ -33,6 +33,7 @@ import Clash.Prelude (withClockResetEnable)
 import Clash.Explicit.Prelude
 import qualified Clash.Explicit.Prelude as E
 
+import qualified Data.Map as Map (singleton)
```

Comment:
For Bittide we decided to put all the qualified imports at the very bottom to keep lists trivially sortable